### DISCUSSION TITLE
Where to get code to reproduce the benchmarks from https://github.com/pyannote/pyannote-audio ?

### DISCUSSION THREAD

**Discussion by timotheecour** [pyannote/pyannote-audio#1643] [Q&A] [answered]
The benchmark section in the readme of https://github.com/pyannote/pyannote-audio shows results for a number of datasets; is there code to reproduce those numbers (at least code to load data + groundtruth for each dataset, since each dataset seems to have their own format)

**Comment by hbredin** [comment, marked as answer]
@FrenchKrab started [this](https://github.com/FrenchKrab/datasets-pyannote) a while back.
There is also [this](https://github.com/pyannote/AMI-diarization-setup/) and [this](https://github.com/BUTSpeechFIT/CALLHOME_sublists). 
You might also want to look at [lhotse](https://github.com/lhotse-speech/lhotse).
